addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2668,18 +2668,118 @@ options.registry.CookiesBar = options.registry.SnippetPopup.extend({
     },
 });
 
+/**
+ * Base class for all cover updaters.
+ */
+const CoverUpdaterMixin = (parentClass) => {
+    const Extended = parentClass.extend({
```

Comment:
Strange way to do it :grimacing: Can't you do the conventional way and do `.extend(CoverUpdateMixin)` everywhere you want to use it ? (the mixin being a simple object ?)